core/tests/puppeteer-acceptance-tests/spec/logged-in-user-tests/click-more-links-in-oppia-footer.spec.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for checking if logged-in users can
+ * follow links in the oppia footer not covered by other tests.
+ */
+
+import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
+import {LoggedInUser} from '../../user-utilities/logged-in-users-utils';
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
+
+const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
+
+describe('Logged-in User', function () {
+  let testUser: LoggedInUser;
+
+  beforeAll(async function () {
+    testUser = await UserFactory.createNewUser(
+      'testuser',
+      'testuser@example.com'
+    );
+  }, DEFAULT_SPEC_TIMEOUT_MSECS);
+
+  beforeEach(async function () {
+    // Navigate to a page that has the oppia footer.
+    await testUser.navigateToAboutPage();
+  });
+
+  it(
+    'should open the "Volunteer" page via the footer',
+    async function () {
+      await testUser.clickOnVolunteerLinkInFooter();
+    },
+    DEFAULT_SPEC_TIMEOUT_MSECS
+  );
+
+  it(
+    'should open "Donate" page via the footer',
+    async function () {
+      await testUser.clickOnDonateLinkInFooter();
+    },
+    DEFAULT_SPEC_TIMEOUT_MSECS
+  );
+
+  it(
+    'should open "Community Library" page via the footer',
+    async function () {
+      await testUser.clickOnBrowseTheLibraryLinkInFooter();
+    },
+    DEFAULT_SPEC_TIMEOUT_MSECS
+  );
+
```

Comment:
@seanlip @agallop There's one more button in the footer. It's the "Get it on Google Play" button. Should we also check on that button since it's also inside the footer?

Reply:
Yeah, might as well, I think?

Reply:
Done. in the social links test.